launchmon/src/linux/sdbg_linux_ptracer_impl.hxx

```diff
@@ -119,9 +120,30 @@ linux_ptracer_t<SDBG_DEFAULT_TEMPLPARAM>::tracer_setregs (
     }
 
   regset->set_ptr_to_regset();
-  num_regs = regset->size_in_word();
-  offset = regset->get_offset_in_user();
 
+#if AARCH64_ARCHITECTURE
+
```

Comment:
I can take this as is. But it will be nice if we at some point test whether this change can be used for all other platforms so that we don' have to use if/def.